src/com/lftechnology/java/training/dipak/guess/GuessTheNumber.java

```diff
@@ -0,0 +1,84 @@
+package com.lftechnology.java.training.dipak.guess;
+
+import java.util.Scanner;
+import java.util.logging.Logger;
+
+/*Guessing a number game application: 
+*application will generate a random no between 1 and 20. 
+*user guess the value by entering from the console
+*Check if the guess value matched with random number or not, if matched show the guessing value, random value and the no. of attempts. 
+ */
+
+public class GuessTheNumber{
+	private final static Logger LOGGER = Logger.getLogger(GuessTheNumber.class.getName());
+	private int theNumber;
+	private int attempts=0;
+	private int lowerBound;
+	private int upperBound;
+	
+	GuessTheNumber(int lowerBound,int upperBound){
+		this.lowerBound = lowerBound;
+		this.upperBound = upperBound;
+	}
+	/**
+	 * <p>This method generates the random number to be guessed.</p>

```

Comment:
Generates the random number to be guessed would be suitable for the description.